bee-common/bee-common/src/packable/mod.rs

```diff
@@ -0,0 +1,209 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! A module that provides a `Packable` trait to serialize and deserialize types.
+
+#[cfg(feature = "alloc")]
+mod alloc;
```

Comment:
Why is this under a feature btw ? no_std libraries can depend on alloc

Reply:
is `alloc` always available when you use `no_std`?

Reply:
From the doc `While #![no_std] binaries using alloc still require nightly Rust, #![no_std] library crates can use the alloc crate in stable Rust.` https://doc.rust-lang.org/edition-guide/rust-next/alloc.html

Reply:
great :D